tests/generate.py

```diff
@@ -159,9 +159,13 @@ def main():
         whitelist = set(sys.argv[1:])
 
     if platform.system() == "Windows":
-        asyncio.set_event_loop(asyncio.ProactorEventLoop())
+        asyncio.get_event_loop_policy().set_event_loop(asyncio.ProactorEventLoop())
```

Comment:
Isn't manually instanciating an event loop discouraged? 
```suggestion
        asyncio.set_event_loop_policy(asyncio.WindowsProactorEventLoopPolicy())
```

Reply:
Ah that is a better fix.